lib/API.jsx

```diff
@@ -327,6 +327,20 @@ export default function API(network, args) {
             return performPOSTRequest(commandName, parameters);
         },
 
+        /**
+         * Initiate a screen share request with a user.
+         *
+         * @param {object} parameters
```

Comment:
I believe we should make this upper-case

```suggestion
         * @param {Object} parameters
```